lib/Macros/Sources/SwiftMacros/SwiftifyImportMacro.swift

```diff
@@ -700,370 +700,450 @@ func getParameterIndexForDeclRef(
   return try getParameterIndexForParamName((parameterList), ref.baseName)
 }
 
-/// A macro that adds safe(r) wrappers for functions with unsafe pointer types.
-/// Depends on bounds, escapability and lifetime information for each pointer.
-/// Intended to map to C attributes like __counted_by, __ended_by and __no_escape,
-/// for automatic application by ClangImporter when the C declaration is annotated
-/// appropriately. Moreover, it can wrap C++ APIs using unsafe C++ types like
-/// std::span with APIs that use their safer Swift equivalents.
-public struct SwiftifyImportMacro: PeerMacro {
-  static func parseEnumName(_ expr: ExprSyntax) throws -> String {
-    var exprLocal = expr
-    if let callExpr = expr.as(FunctionCallExprSyntax.self) {
-      exprLocal = callExpr.calledExpression
-    }
-    guard let dotExpr = exprLocal.as(MemberAccessExprSyntax.self)
-    else {
-      throw DiagnosticError(
-        "expected enum literal as argument, got '\(expr)'",
-        node: expr)
-    }
-    return dotExpr.declName.baseName.text
+func parseEnumName(_ expr: ExprSyntax) throws -> String {
```

Comment:
Factoring these noop changes (moving methods out to be free functions) out into a separate PR would make reviewing this one with the functional changes much easier.

Reply:
These have been merged in #80400 now.